.gitignore

```diff
@@ -13,11 +13,5 @@ bin
 # Dirty file in OS X
 .DS_Store
 
-# Config when used with VS Code
-.vscode/
```

Comment:
Why ?

Reply:
Global configuration can avoid `.vscode` for each project.

Reply:
But we can't assume everyone have this global configuration. And any problem if we configure it again ?

Reply:
Agree with cd1989